axis_backend/apps/documents/views/document_viewset.py

```
@@ -0,0 +1,511 @@
+            request: HTTP request with optional days parameter
+
+        Returns:
+            Response with expiring documents
+        """
+        days = int(request.query_params.get('days', 30))
+
+        try:
+            documents = self.service.get_expiring_documents(days=days)
+            serializer = self.list_serializer_class(documents, many=True)
+            return Response(serializer.data)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)